src/uwtools/tests/drivers/test_mpas.py

```diff
@@ -0,0 +1,233 @@
+from pathlib import Path
+from unittest.mock import DEFAULT as D
+from unittest.mock import patch
+
+import f90nml  # type: ignore
+import pytest
+import yaml
+from pytest import fixture, raises
+
+from uwtools.drivers import mpas
+from uwtools.exceptions import UWConfigError
+from uwtools.tests.support import fixture_path
+
+# Fixtures
+
+
+@fixture
+def cycle():
+    return dt.datetime(2024, 3, 22, 6)
+
+
+# Driver fixtures
+
+
+@fixture
+def config(tmp_path):
+    return {
+        "mpas": {
+            "execution": {"executable": "atmosphere_model"},
+            "lateral_boundary_conditions": {
+                "interval_hours": 1,
+                "offset": 0,
+                "path": str(tmp_path / "input_files"),
+            },
+            "length": 1,
+            "namelist": {
+                "base_file": str(fixture_path("simple.nml")),
+                "update_values": {
+                    "nhyd_model": {"config_start_time": "12", "config_stop_time": "12"},
+                },
+            },
+            "run_dir": str(tmp_path),
+            "streams": {
+                "path": str(tmp_path / "streams.atmosphere.in"),
+                "values": {
+                    "world": "user",
+                },
+            },
+            "files_to_link": {
+                "CAM_ABS_DATA.DBL": "src/MPAS-Model/CAM_ABS_DATA.DBL",
+                "CAM_AEROPT_DATA.DBL": "src/MPAS-Model/CAM_AEROPT_DATA.DBL",
+            },
+        }
+    }
+
+
+@fixture
+def config_file(config, tmp_path):
+    path = tmp_path / "config.yaml"
+    with open(path, "w", encoding="utf-8") as f:
+        yaml.dump(config, f)
+    return path
+
+
+@fixture
+def driverobj(config_file, cycle):
+    return mpas.MPAS(config=config_file, cycle=cycle, batch=True)
+
+
+# Driver tests
+
+
+def test_MPAS(driverobj):
+    assert isinstance(driverobj, mpas.MPAS)
+
+
+def test_MPAS_boundary_files(driverobj, cycle):
+    ns = (0, 1)
+    links = [
+        driverobj._rundir / f"lbc.{(cycle+dt.timedelta(hours=n)).strftime('%Y-%m-%d_%H.%M.%S')}.nc"
+        for n in ns
+    ]
+    assert not any(link.is_file() for link in links)
+    infile_path = Path(driverobj._driver_config["lateral_boundary_conditions"]["path"])
+    infile_path.mkdir()
+    for n in ns:
+        (
+            infile_path / f"lbc.{(cycle+dt.timedelta(hours=n)).strftime('%Y-%m-%d_%H.%M.%S')}.nc"
+        ).touch()
+    driverobj.boundary_files()
+    assert all(link.is_symlink() for link in links)
+
+
+def test_MPAS_dry_run(config_file, cycle):
+    with patch.object(mpas, "dryrun") as dryrun:
+        driverobj = mpas.MPAS(config=config_file, cycle=cycle, batch=True, dry_run=True)
+    assert driverobj._dry_run is True
+    dryrun.assert_called_once_with()
+
+
+@pytest.mark.parametrize(
+    "key,task,test",
+    [("files_to_copy", "files_copied", "is_file"), ("files_to_link", "files_linked", "is_symlink")],
+)
+def test_MPAS_files_copied(config, cycle, key, task, test, tmp_path):
```

Comment:
This method tests both copying and linking files, so its name should probably be more general. Ditto for the corresponding test in the `MPASInit` tests. And ditto for the FV3 test that these are based on, which should have had a better name to begin with (my fault). Maybe even just more explicit like `test_*_files_copied_and_linked`.